add fedora information readme

mencattini opened the pull request:
Ajout des informations pour la partie Fedora

saucisson commented:
I have updated the README, and added several files. Can you please update your fork using the current README?

mencattini commented:
J'ai update le fork à partir de cui-unige/modeling-verification.

Est-ce que je dois refaire un pull request?


Le 03. 10. 16 à 22:41, Alban Linard a écrit :
>
> I have updated the README, and added several files. Can you please 
> update your fork using the current README?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/cui-unige/modeling-verification/pull/1#issuecomment-251220435>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AQCrObBJ_yHPrz37VBUZ4jrIp1TrJV2nks5qwWh3gaJpZM4KKJwr>.
>

saucisson commented:
In order to handle several variants, and to allow their testing, i have updated again the repository.
To add a system variant, you now need to:
* fill the `case ... esac` part of the `bin/install` in the `master` branch;
* update `README.md` to add the supported virtual machine in the `master` branch;
* create a branch with the system name (for instance `fedora`), and update the `box:` line in `wercker.yml` to match a [dockerhub](https://hub.docker.com) image for you system (for instance `fedora:latest` as seen on https://hub.docker.com/_/fedora/); this branch must contain the latest changes in `master`, so you can use `git merge master` to update it whenever `master` is changed.

You can test if your additions are correct by running `wercker build` in your branch. Of course, you need to install [wercker](http://wercker.com/cli/) on your system.
